policy/redhat/5/test_ipc.te

```diff
@@ -77,7 +77,7 @@ corecmd_bin_entry_type(ipcdomain)
 userdom_sysadm_bin_spec_domtrans_to(ipcdomain)
 
 allow test_ipc_base_t self:sem create_sem_perms;
-allow test_ipc_base_t self:shm create_sem_perms;
+allow test_ipc_base_t self:shm create_shm_perms;
```

Comment:
Doesn't this fix also allow us to remove the separate allow line for lock permission?

Reply:
Hm, indeed... Maybe it wasn't a typo and `create_shm_perms` just didn't exist in the past? Either way, I'll remove the superfluous lines and update the commit message.